internal/server/keyRingEndpoint.go

```diff
@@ -46,6 +46,15 @@ func (kre *KeyRingEndpoint) AttachHandlers(r *mux.Router) {
 	r.HandleFunc("/deletePrivateKey", kre.deletePrivateKey).Methods("POST")
 }
 
+// Get GPG Key godoc
+// @id kre-get-key
+// @tags Key Ring
+// @Summary Fetches a public key from the server
```

Comment:
Tem algum servidor específico? É o mesmo `Keyserver` mencionado acima?

```Fetches a Public GPG Key from the server```

Reply:
Boa, não é. Acho que dá pra omitir o `from the server` pois ele faz o fetch do proprio remote-signer.